manifests/kustomize/README.md

```diff
@@ -81,7 +75,25 @@ Or you can add it in one line with:
 
 ```bash
 kubectl get configmap centraldashboard-config -n kubeflow -o json | jq '.data.links |= (fromjson | .menuLinks += [{"icon": "assignment", "link": "/model-registry/", "text": "Model Registry", "type": "item"}] | tojson)' | kubectl apply -f - -n kubeflow
-````
+```
+
+#### Installing Model Registry on a Kubeflow Profile
+
+Kubeflow Central Dashboard uses [Profiles](https://www.kubeflow.org/docs/components/central-dash/profiles/) to handle user namespaces and permissions. By default, the manifests deploy the Model Registry instance in the `kubeflow` namespace, to install a compatible version of Model Registry for Kubeflow, you should deploy a separate instance of Model Registry in the profile's namespace. For that just run:
+
+```shell
+cd options/istio
+kustomize edit set namespace <your-profile>
```

Comment:
is same as "namespce"? I did not think I had to provide any profile name (that I remember) when installing central dashboard. What the default or we can be explicit here what to provide?

Reply:
As stated in the docs linked above "A Kubeflow Profile is a [Kubernetes CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) introduced by Kubeflow that wraps a Kubernetes [Namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/)." and "Kubeflow supports automatic profile creation for users who log into Kubeflow for the first time." users can infer from the docs that when deploying a KCD instance, a namespace is created for the user, instructions shouldn't rely on other docs to avoid duplications.

Reply:
If we duplicate the instructions from here from the linked doc, we're introducing the risk of divergence over time. Anyone updating the core kubeflow docs is not going to be aware they need to update the MR instructions as well, unless they happen to be very familiar with the MR manifests.